config/php-fpm/Dockerfile

```diff
@@ -51,6 +50,12 @@ COPY composer/config.json /var/www/.composer/config.json
 # Copy xdebug and php config.
 COPY conf.d/* /usr/local/etc/php/conf.d/
 
+# Fix permissions and umask
+RUN chown -R www-data /var/www
```

Comment:
i am not sure if these permissions are best practice. but something like this is needed in order that the www-data user is allowed to write the `var/www/.bash_history` file

Reply:
why not set the persmission only on the file?